### ISSUE TITLE
Use the same file structure for local modules as for nf-core modules

### ISSUE THREAD

**Issue by krokicki** [nf-core/tools#1929]
### Description of feature

Currently local modules have a single file structure e.g. `local</tool>/<subtool>.nf`. It would be nice if the structure mirrored the nf-core module structure e.g. `local/<tool>/<subtool>/main.nf` and `meta.yml`. This would make it easier to promote local modules into nf-core modules.

**Comment by yocra3** [comment]
I totally agree with this issue. I think that the tool should also create the test files. This issue is related with #1947

**Comment by mirpedrol** [comment]
Done in https://github.com/nf-core/tools/pull/3256 🎉
